<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVLineLoader_Widget</class>
 <widget class="QWidget" name="CSVLineLoader_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>CSV Line Data Loading</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>Load line data from CSV files. Choose between single file (all timestamps) or multi-file (one file per timestamp) format.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="load_mode_groupbox">
     <property name="title">
      <string>Load Mode</string>
     </property>
     <layout class="QVBoxLayout" name="load_mode_layout">
      <item>
       <widget class="QRadioButton" name="single_file_radio">
        <property name="text">
         <string>Single File - All timestamps in one CSV file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="multi_file_radio">
        <property name="text">
         <string>Multi File - One CSV file per timestamp</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_file_path">
       <property name="text">
        <string>File/Directory:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="file_path_edit">
       <property name="placeholderText">
        <string>Select file or directory...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="browse_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_delimiter">
       <property name="text">
        <string>Delimiter:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="delimiter_combo">
       <item>
        <property name="text">
         <string>Comma</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Space</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tab</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_x_column">
       <property name="text">
        <string>X Column:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSpinBox" name="x_column_spinbox">
       <property name="toolTip">
        <string>Column index for X coordinates (0-based). Only used for multi-file mode.</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_y_column">
       <property name="text">
        <string>Y Column:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QSpinBox" name="y_column_spinbox">
       <property name="toolTip">
        <string>Column index for Y coordinates (0-based). Only used for multi-file mode.</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QCheckBox" name="has_header_checkbox">
       <property name="text">
        <string>Files have header row</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_coordinate_delimiter">
       <property name="text">
        <string>Coordinate Delimiter:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="coordinate_delimiter_combo">
       <property name="toolTip">
        <string>Delimiter used within X and Y coordinate lists. Only used for single-file mode.</string>
       </property>
       <item>
        <property name="text">
         <string>Comma</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Space</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tab</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_header_identifier">
       <property name="text">
        <string>Header Identifier:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="header_identifier_edit">
       <property name="toolTip">
        <string>Text in the first column that identifies the header row. Only used for single-file mode.</string>
       </property>
       <property name="text">
        <string>Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="load_button">
     <property name="text">
      <string>Load CSV Data</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
